### PR TITLE
feat: save columns with localStorage

### PR DESCRIPTION
**Pull request by abbesAlexandre**
Feat : add saving column with localstorage

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -3,7 +3,22 @@
 function tagElementClick() {
   this.classList.toggle("selected");
 
+  let config = JSON.parse(localStorage.getItem("config"));
```
Comment (PierreDemailly) [inline public/js/main.js:6]:
```suggestion
  let config = JSON.parse(localStorage.getItem("hidden-tags"));
```
Or something like that, what do you think ? It feel `config` is too generic

**Review by fraxken — APPROVED** [review]
LGTM

File: public/js/main.js [hunk 2]
```diff
@@ -268,6 +283,17 @@ document.addEventListener("DOMContentLoaded", () => {
 
       buildOrglist();
       initListJS();
+
+      const configArray = JSON.parse(localStorage.getItem("config"));
+      if (configArray !== null || configArray.length !== 0) {
```
Comment (PierreDemailly) [inline public/js/main.js:288]: It throws
> main.js:288 Uncaught TypeError: Cannot read properties of null (reading 'length')
    at WebSocket.<anonymous>
 
if config is empty.
`if (configArray?.length > 0)`should do the job

**Review by PierreDemailly — APPROVED** [review]

**Comment by PierreDemailly** [comment]
@allcontributors please add @abbesAlexandre for code

**Pull request merged**